docs/getting_started.rst

```diff
@@ -81,34 +95,26 @@ Detailed steps (manual environment setup):
       happen)
     * Run ``PYCURL_SSL_LIBRARY=nss pip install -U -r requirements.txt --no-cache-dir``
 
-* Copy template files in cfme_tests to the same file name without the ``.template`` extension
-
-  * Example: ``cp file.name.template file.name``
-  * Bash script example: ``for file in *.template; do cp -n $file ${file/.template}; done``
-  * Edit these files as needed to reflect your environment.
-
-* Do the same for the config yamls in the conf directory, using Configuration YAMLs
-
-  * Example: ``cd conf/; cp env.local.template env.local``
-  * Then edit ``conf/env.local.yaml`` to override ``base_url``
-
+* You copy/symlink the required YAML files into ``conf/`` if you have access to team's internal YAML
+  repository. Required YAML files are ``env``, ``cfme_data``, ``credentials``. If the file's
+  extension is ``.yaml`` it is loaded normally, if its extension is ``.eyaml`` then it is encrypted
+  and you need to have the decryption key in the ``cfme_tests/`` directory. You can also start them
+  from scratch by copying the templates in ``conf/`` and editing them to suit the environment you
+  use.
 * Set up a local selenium server that opens browser windows somewhere other than your
-  desktop by running :doc:`guides/vnc_selenium`
+  desktop by running :doc:`guides/vnc_selenium`. This is the preferred approach.
```

Comment:
I would say preferred is to use dockerbot ```sel_container``` script, because then you are using a "known" browser.